data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg

```diff
@@ -865,7 +715,7 @@ We would be wise to secure all settlements before then."
 
 I have found anoþer Aquadyne. It is here, deep beneaþ Þe Þree Sisters. It has only been þree years since Haldric destroyed þe elf gosts þat were sapping it and already þere is a mælstrom formed off þe norþ shore. Just like þose mælstroms Seeker Perþ discovered west of Alduin before a Wisp chased away his ship.
 
-Indeed, þere must be an Aquadyne every twenty leagues around þe entire Great Continent. Everyþing is just as Seeker Oþila had predicted. We should never have left her behind on þe Green Isle. Wiþout her work it may take centuries before we of þe Silver Order know what mankind is dealing wiþ on þis continent. Þe Færie Realm is stronger here þan we ever imagined.
+Indeed, þere must be an Aquadyne every twenty leagues around þe entire Great Continent. Everyþing is just as Seeker Oþila had predicted. We should never have left her behind on þe Green Isle. Wiþout her work it may take centuries before we of þe Silver Order know what mankind is dealing wiþ on þis continent. Þe Færie World is stronger here þan we ever imagined.
```

Comment:
What's the reasoning for world vs. realm? We don't actually have fleshed out lore for either, but I want to get your thoughts.

Reply:
The wose unit description refers to it as "faerie world" but somehow I misremembered it as 'realm' when I first wrote this. So the aim here is just better consistency with the wider lore and fewer synonyms for the player to try to remember.